ui/pages/Project/components/RnaSeqResultPage.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Loader, Grid } from 'semantic-ui-react'
+
+import { getGenesById, getSamplesByGuid, getRnaSeqDataByIndividual, getRnaSeqSignificantJunctionData, getIndividualsByGuid } from 'redux/selectors'
+import { RNASEQ_JUNCTION_PADDING } from 'shared/utils/constants'
+import DataLoader from 'shared/components/DataLoader'
+import { loadRnaSeqData } from '../reducers'
+import { getRnaSeqDataLoading } from '../selectors'
+
+const RnaSeqOutliers = React.lazy(() => import('./RnaSeqOutliers'))
+const RnaSeqOutliersTable = React.lazy(() => import('./RnaSeqJunctionOutliersTable'))
+
+const OUTLIER_VOLCANO_PLOT_CONFIGS = {
+  outliers: {
+    getLocation: (({ geneId }) => geneId),
+    searchType: 'genes',
+    title: 'OUTRIDER Outliers',
```

Comment:
We should either say "OUTRIDER" or "Expression Outliers" but "OUTRIDER Outliers" is redundant. Ditto for Fraser